src/sections/FAQ.tsx

```diff
@@ -0,0 +1,30 @@
+import Question from "../components/Question";
+import { faq_questions } from "../data/questionData";
```

Comment:
```suggestion
import Question from "~/components/Question";
import { faq_questions } from "~/data/questionData";
```
Please always use absolute paths